cloud-gcp/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     implementation group: 'com.dyngr',                     name: 'polling',                                    version: dyngrPollingVersion
     implementation group: 'org.slf4j',                     name: 'slf4j-api',                                  version: slf4jApiVersion
     implementation group: 'org.apache.commons',            name: 'commons-lang3',                              version: apacheCommonsLangVersion
+    implementation group: 'org.jetbrains',                 name: 'annotations',                                version: '23.1.0'
```

Comment:
could you remove this?

Reply:
So should I remove the @NotNull annotations as well? eg. https://github.com/hortonworks/cloudbreak/blob/43c1b0ccc3d7e5cbf78de89bf6b42d7f26f4e810/common/src/main/java/com/sequenceiq/cloudbreak/logger/RestLoggerFilter.java#L78

Reply:
right, those intellij annotations shouldn't be there at all

Reply:
we might need a checkstyle rule to forbid intellij annotations